pkg/objstore/inmem/inmem.go

```diff
@@ -57,23 +84,31 @@ func (b *Bucket) Iter(_ context.Context, dir string, f func(string) error) error
 
 // Get returns a reader for the given object name.
 func (b *Bucket) Get(_ context.Context, name string) (io.ReadCloser, error) {
+	if name == "" {
+		return nil, errors.New("inmem: object name is empty")
+	}
+
 	file, ok := b.objects[name]
 	if !ok {
-		return nil, errors.Errorf("no such file %s", name)
+		return nil, errNotFound
```

Comment:
I like having filename inside the error message :), can we keep it?

Reply:
I can, sure, just wanted to keep it simple (I need to assert not found err in tests). If not I would endup doing some `IsNotFoundErr` magic. Can do if you want.

Reply:
especially it is inmem ONLY used in tests.. let's just leave it simple, what do you think?

Reply:
sgtm